pydy/viz/server.py

```diff
@@ -5,12 +5,16 @@
 from SimpleHTTPServer import SimpleHTTPRequestHandler
 
 
-def run_server(port=8000,scene_file="Null"):
-    #change dir to static first.
-    os.chdir("static/")
+def run_server(port=8000, scene_file="Null"):
+    # change dir to static first.
+    if os.path.exists(os.path.join('static', scene_file)):
```

Comment:
I don't think we need this either. The server should always run the viz from the files in the source tree.